oragono.yaml

```diff
@@ -159,6 +159,14 @@ accounts:
     # is account authentication enabled?
     authentication-enabled: true
 
+    # will the server enforce that only the account holder can use the account name as a nick?
+    # options:
+    # `disabled`: no enforcement
+    # `timeout` (auth to nickserv within some period of time or you're disconnected)
+    # `strict`: must authenticate up front with SASL
+    nick-reservation: disabled
+    nick-reservation-timeout: 30s
```

Comment:
We should make this a section, like:

```
nick-reservation:
    enabled: false
    timeout: 30s
    rename-prefix: Guest-
```
I'd like to make the 'timeout' penalty be that they're renamed to, for instance, `Guest-92h4gu2b` using the prefix like the above. That matches what other servers do and prevents AFK users from getting put into a loop of being disconnected and then reconnecting, over and over again.

Reply:
Sounds good to me! Would you mind copying this into #169?